Modules/services/Model/SeTask.php

```diff
@@ -160,7 +160,7 @@ public function getTaskServicesIds($id_space, $id_task) {
 
     public function setTaskService($id_space, $id_task, $id_service) {
         if ($this->isTaskService($id_space, $id_task, $id_service)) {
-            $sql = "UPDATE se_task_service (id_space, id_task, id_service) VALUES (?, ?, ?)";
+            $sql = "UPDATE se_task_service SET id_space=?, id_task=?, id_service=?;";
```

Comment:
Why a  set vs previous update. This is ok, just wondering why

Reply:
I think I had a problem testing with sql in strict mode. But don't see why there would have been a problem on this line...